src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double areaCalculation();
```

Comment:
Nethod names should be verbs that give us understanding of the result of method call.
Nouns are siutable for classes/interfaces names
```suggestion
    double getArea();
```